recipes/libcufile/Makefile.patch

```diff
@@ -0,0 +1,14 @@
+--- samples/Makefile        2022-06-24 00:23:23.000000000 +0000
++++ samples/Makefile.new    2022-06-27 15:04:32.235125647 +0000
+@@ -35,8 +35,8 @@
+ 
+ # Common includes and paths for CUDA
+ CUDA_PATH   := /usr/local/cuda
+-CUFILE_PATH ?= /usr/local/cuda/targets/x86_64-linux/lib/
+-CUFILE_INCLUDE_PATH ?= /usr/local/cuda/targets/x86_64-linux/include/
++CUFILE_PATH ?= /usr/local/cuda/lib/
++CUFILE_INCLUDE_PATH ?= /usr/local/cuda/include/
```

Comment:
@adibbley, wondering if these should be pointing to the Conda environment. Maybe something like this (though tweaks could be needed

```suggestion
-CUDA_PATH   := /usr/local/cuda
-CUFILE_PATH ?= /usr/local/cuda/targets/x86_64-linux/lib/
-CUFILE_INCLUDE_PATH ?= /usr/local/cuda/targets/x86_64-linux/include/
+CUDA_PATH   := ${PREFIX}
+CUFILE_PATH ?= $(CUDA_PATH)/lib/
+CUFILE_INCLUDE_PATH ?= $(CUDA_PATH)/include/
```

Reply:
Agreed it should be pointing to the Conda environment, we could also add the `targets` directory back now that we have that here:
```suggestion
 CUDA_PATH   := ${PREFIX}
-CUFILE_PATH ?= /usr/local/cuda/targets/x86_64-linux/lib/
-CUFILE_INCLUDE_PATH ?= /usr/local/cuda/targets/x86_64-linux/include/
+CUFILE_PATH ?= $(CUDA_PATH)/targets/x86_64-linux/lib/
+CUFILE_INCLUDE_PATH ?= $(CUDA_PATH)/targets/x86_64-linux/include/
```